github/test_download_s3.py

```diff
@@ -0,0 +1,72 @@
+"""
+make sure we can download a file on demand
+"""
+import resources as r
+import pyjq
+import pytest
+
+
+
+@pytest.mark.github_org
+@pytest.mark.rationale(
+        """
+Sanity check for early development
```

Comment:
Is this entire file to be removed?

Reply:
I'll rephrase the comment -- originally yes, but there is so much scaffolding outside the pytest for file delivery and here I consume _our_ JSON files, which are not as strongly versioned as AWS/GCP, so I'm going to leave them in.